flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/data/TestFlinkParquetReader.java

```diff
@@ -199,6 +204,222 @@ public void testTwoLevelList() throws IOException {
     }
   }
 
+  @Test
+  public void testMissingRequiredWithoutDefault() {
```

Comment:
The other tests added `supportsDefaultValues` and the new cases to [`DataTest`](https://github.com/apache/iceberg/blob/main/data/src/test/java/org/apache/iceberg/data/parquet/TestGenericData.java#L99-L102), which I think this inherits from. Why not use the same test cases? Were there changes?

Reply:
Let me check. I mimic what's in Spark (maybe it changed recently).

Reply:
Ah yeah, a while ago I think @jbonofre and I were talking and I pointed to the Spark PR for examples of tests for default values but it seems like it got later refactored to move those to `DataTest`.

@jbonofre I think all we'd need to do here is just override supportsDefaultValues to just return true.

Reply:
Yes, that's right, it changed between the time I started the PR and now. No worries at all, let me reuse `supportsDefaultValues` here. Thanks !